common/frame.go

```diff
@@ -746,15 +746,28 @@ func (f *Frame) Content() string {
 	f.log.Debugf("Frame:Content", "fid:%s furl:%q", f.ID(), f.URL())
 
 	rt := f.vu.Runtime()
-	js := `let content = '';
-		if (document.doctype) {
-			content = new XMLSerializer().serializeToString(document.doctype);
-		}
-		if (document.documentElement) {
-			content += document.documentElement.outerHTML;
-		}
-		return content;`
-	return f.Evaluate(rt.ToValue(js)).(string)
+	js := `() => {
+			let content = '';
+			if (document.doctype) {
+				content = new XMLSerializer().serializeToString(document.doctype);
+			}
+			if (document.documentElement) {
+				content += document.documentElement.outerHTML;
+			}
+			return content;
+		}`
+
+	c := f.Evaluate(rt.ToValue(js))
+	if c == nil {
+		return ""
+	}
```

Comment:
Why is this `nil` check necessary here? Can it mask errors?

I was suspicious of it. Then, I removed it, and the test still passed.

Reply:
It follows the same pattern as `InnerHTML()` and `InnerText()`. I suppose there are cases where it could be `nil`, so this is just a safeguard.

Reply:
I don't think it's necessary and can't think of how it may occur. If you insist, would it be better to add a log if it's nil (error level)? Or get rid of it until a user complains so we can find why it happens?